src/engine/strat_engine/pool.rs

```diff
@@ -41,12 +41,12 @@ use super::metadata::MIN_MDA_SECTORS;
 use super::serde_structs::{FilesystemSave, FlexDevsSave, PoolSave, Recordable, ThinPoolDevSave};
 use super::setup::{get_blockdevs, get_metadata};
 
-const DATA_BLOCK_SIZE: Sectors = Sectors(2048);
+pub const DATA_BLOCK_SIZE: Sectors = Sectors(2048);
 const META_LOWATER: u64 = 512;
 const DATA_LOWATER: DataBlocks = DataBlocks(512);
-
+const DATA_EXPANSION_PERCENT: f32 = 20.0;
```

Comment:
It's not the amount I object to, it's the use of percentages at all, and the use of floats to represent percentages.

Shoving the calculation into a function with a TODO seems like the right thing to do. But please leave out any use of percentages or floats. Doubling is just the simplest thing to do, so I'ld recommend it.